packages/backend/src/routers/projectRouter.ts

```diff
@@ -183,6 +183,37 @@ projectRouter.post(
     },
 );
 
+projectRouter.get(
+    '/field/:fieldId/search',
+    isAuthenticated,
+    async (req, res, next) => {
+        try {
+            const value: string =
+                typeof req.query.value === 'string'
+                    ? req.query.value.toString()
+                    : '';
+            const limit: number =
+                typeof req.query.limit === 'string'
+                    ? parseInt(req.query.limit.toString(), 10)
```

Comment:
why do you need to do this verbosy type checking ?

Reply:
what part ? checking the type, toString, parseInt
Just based myself on existing code from another endpoint with url params. 
The `toString` might be unnecessary.